functions-methods.js

```diff
@@ -33,4 +74,18 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+// function(checkEmailValidity){
+//   return
+// }
+function checkEmailValidity(email) {
+  var re = /\S+@\S+\.\S+/;
+  return re.test(email);
```

Comment:
var binnen javascript is deprecated. Gebruik let of const. Je gebruikt over let en const maar hier opeens var, AI??

Verder is de bedoeling dat je zelf de logica schrijft en geen regex gebruikt.